tahoma_api/client.py

```diff
@@ -141,6 +141,16 @@ async def get_current_executions(self) -> List[Any]:
 
         return response
 
+    async def get_scenarios(self) -> List[Scenario]:
+        """ List the scenarios """
+        response = await self.__get("actionGroups")
```

Comment:
I can understand that we don't need humps here, since it would slow down the request. However this will cause us having camelCase and snake_case throughout this file. Do we prefer millisecond performance hit over readability? Would be good to think about this for future PR's.

Reply:
For the moment, we only care about label and oid. So there is no need to transform the response :)

Reply:
@tetienne nevermind, my reference was to `actionGroups`, however I missed that this is the GET endpoint...